### ISSUE TITLE
Current style looks a bit off in new persona-theme

### ISSUE THREAD

**Issue by rbeer** [Linux-statt-Windows/nodebb-plugin-topic-events#5]
@niklas-heer 
The standard design is very little distinguishable from post entries, now in the new persona-theme.
I tried to center them between the posts with `margin-bottom: 20px`, but that leads to ripping consecutive events (without a post inbetween) apart. :weary: 
Boxing them in another div or something, like I did in 0.1, isn't an option anymore.

![persona-flaw](https://cloud.githubusercontent.com/assets/3411649/7983558/e0146692-0ac1-11e5-91a6-f87d3b714d18.png)

**Comment by niklas-heer** [comment]
> Boxing them in another div or something, like I did in 0.1, isn't an option anymore.

Why is that?

**Comment by rbeer** [comment]
Well, I actually just decided against it. For one, because the re-ordering on that infinite scroll would require re-packing of the events:

``` html
... posts above here not loaded, yet ...
<event-wrap>
  <event-after#2 />
  <event-after#3 />
</event-wrap>
<post#4 />
<post#5 />
```

`scrolling up`

``` html
<post#1 />
<post#2 />
<event-wrap>
  <event-after#2 />
</event-wrap>
<post#3 />
<event-wrap>
  <event-after#3 />
</event-wrap>
<post#4 />
<post#5 />
```

The second reason is, because I am trying not to interfere with any other plugins, that might want to add Elements based on their timestamp in the topic-component. While this is not yet exactly the case, I know, boxing might complicate things further, there.

``` html
<post time="13.10" />
<event-wrap>
  <event time="13.15" />
... some other plugin might want to add something with e.g. time="13.40" ...
  <event time="13.55" />
<even-wrap>
```

Long story short: I got the feeling, especially due to that infinite scrolling, that the topic-component acts as a stream, constantly receiving and updating elements within. I expect this to become quite popular, that's why I want to keep this as simple as possible.

**Comment by rbeer** [comment]
See, that even helped to figure it out. ^_^
I'm still not very satisfied with the overall looks of it, though.

Then again, it's maybe really just because the name and picture are always the same here. I should try mixing the test topic up a little bit.
![notclear](https://cloud.githubusercontent.com/assets/3411649/8020652/d266bf02-0c80-11e5-923c-60100b75b6c2.png)

**Comment by niklas-heer** [comment]
Mhm okay, but to be honest, i would prefer packaging them in a div and maybe even repackage them, if necessary.
But if you really against that, which I can understand, the only other solution for this problem is to use something like [SASS/Compass mixins](http://thesassway.com/intermediate/if-for-each-while) with conditional statements. 

So you could only apply the `margin-bottom: 20px`, if a post-entry is next to it. But that would be quite some advanced stuff :D

**Comment by rbeer** [comment]
How about the comit? Just adding `padding-top: 20px;` to a post whenever it follows one of `component=topic/event`?

**Comment by niklas-heer** [comment]
It's definitely better than the previous one, but it looks still a bit interrupted by the topic-events. But I think this is the case because of the new persona theme itself, which defines no clear boundaries around the posts.

**Comment by rbeer** [comment]
cleanup/inactivity
